src/neural/factory.cc

```diff
@@ -111,10 +85,15 @@ bool NetworkFactory::BackendConfiguration::operator==(
 
 std::unique_ptr<Network> NetworkFactory::LoadNetwork(
     const OptionsDict& options) {
-  std::string net_path = options.Get<std::string>(kWeightsId);
-  const std::string backend = options.Get<std::string>(kBackendId);
+  std::string net_path =
+      options.Get<std::string>(SharedBackendParams::kWeightsId);
+  const std::string backend =
+      options.Get<std::string>(SharedBackendParams::kBackendId);
   const std::string backend_options =
-      options.Get<std::string>(kBackendOptionsId);
+      options.Get<std::string>(SharedBackendParams::kBackendOptionsId);
+
+  constexpr char* kAutoDiscover = "<autodiscover>";
+  constexpr char* kEmbed = "<built in>";
```

Comment:
The visual stdio compiler on appveyor doesn't like the `constexpr` here.

Reply:
Done, let's see if it compiles. I actually fixed it similarly in one place hoping that another one will fail too in clang, but it didn't.